tests/unit/parts/test_update_metadata.py

```diff
@@ -71,6 +71,12 @@ def yaml_data(extra_args: Dict[str, Any]):
             "base": "core22",
             "grade": "stable",
             "confinement": "strict",
+            "license": "license",
+            "contact": {"contact1"},
```

Comment:
Should these be lists?
```suggestion
            "contact": ["contact1"],
```

Reply:
They can be either simple strings or list, anything due to the added pydentic validator.